docs/zh_cn/01-how-to-build/macos-arm64.md

```diff
@@ -38,7 +38,7 @@
 
 ```bash
 # install pytoch & mmcv-full
-conda install pytorch==1.9.0 torchvision==0.10.0 -c pytorch
+conda install pytorch==1.9.0 torchvision==0.11.0 -c pytorch
```

Comment:
torchvision version shouldn't be updated, should it?

Reply:
yes